site-src/references/policy-attachment.md

```diff
@@ -1,25 +1,38 @@
-# Policy Attachment
-
-While features like timeouts, retries, and custom health checks are present in
-most implementations, their details vary since there are no standards (RFCs)
-around them. This makes these features less portable. So instead of pulling
-these into the API, we offer a middle ground: a standard way to plug these
-features in the API and offer a uniform UX across implementations. This standard
-approach for policy attachment allows implementations to create their own custom
-policy resources that can essentially extend Gateway API.
-
-Policies attached to Gateway API resources and implementations must use the
-following approach to ensure consistency across implementations of the API.
-There are three primary components of this pattern:
-
-* A standardized means of attaching policy to resources.
-* Support for configuring both default and override values within policy
-  resources.
-* A hierarchy to illustrate how default and override values should interact.
-
-This kind of standardization not only enables consistent patterns, it allows
-future tooling such as kubectl plugins to be able to visualize all policies that
-have been applied to a given resource.
+# Metaresources and Policy Attachment
```

Comment:
This is for a separate PR, but reviewing this page reminded me of a broader idea I'd had recently. I think we may want to introduce a broader document/set of documents that cover all the ways the API can be extended. That would include both forms of Policy Attachment as well as ExtensionRefs and GatewayClass Params.